qemu/tests/cfg/block_hotplug.cfg

```diff
@@ -59,6 +59,43 @@
             sub_type_after_plug = boot
             sub_type_after_unplug = boot
             reboot_method = system_reset
+        - with_block_resize:
+            only with_plug
+            sub_type_after_plug = block_resize
+            blk_extra_params_stg0 += " serial=TARGET_DISK0"
+            block_size_cmd = "fdisk -l | grep {0}"
+            block_size_pattern = ",\s+(\d+\s+b)ytes"
+            disk_change_ratio = "1.5 0.5"
+            guest_prepare_cmd = ""
+            Windows:
+                block_size_cmd = wmic diskdrive get size, index
+                block_size_pattern = "1\s+(\d+)"
+                accept_ratio = 0.0005
+                disk_update_cmd = "echo select disk 1 > cmd"
+                disk_update_cmd += " && echo select partition 1 >> cmd"
+                disk_update_cmd += " && echo extend >> cmd"
+                disk_update_cmd += " && echo exit >> cmd"
+                disk_update_cmd += " && diskpart /s cmd"
+                # "::" is used to separate the commands as it is not used in both Linux and Windows
+                disk_update_cmd += "::"
+                disk_update_cmd += "echo select disk 1 > cmd"
+                disk_update_cmd += " && echo select partition 1 >> cmd"
+                disk_update_cmd += " && echo shrink desired=DISK_CHANGE_SIZE >> cmd"
+                disk_update_cmd += " && echo exit >> cmd && diskpart /s cmd"
+                disk_unit = M
+            virtio_scsi:
+                guest_prepare_cmd = "echo 1 > /sys/block/sda/device/rescan"
+                driver_name = vioscsi
+            virtio_blk:
+                driver_name = viostor
+            fmt_qcow2:
+                disk_change_ratio = 1.5
+            fmt_raw:
+                disk_change_ratio = "1.5 0.5"
+            RHEL.5:
+                need_reboot = yes
+            Host_RHEL.m6..ide:
```

Comment:
could you please confirm if ide should set need_reboot=yes?

Reply:
@CongLi, ok, will confirm it later, thank you

Reply:
@CongLi, ide don't support hotplug, so don't need to set params for ide in case: block_hotplug.with_block_resize

Reply:
yes, good catch, but could you please delete it?

Reply:
@CongLi, have deleted, thank you.